resources/queries/cds/nabAntigenWithPanelMeta.sql

```diff
@@ -0,0 +1,40 @@
+SELECT
+ag.assay_identifier,
+ag.antigen_name,
+ag.target_cell,
+ag.antigen_type,
+ag.virus,
+ag.virus_type,
+ag.virus_insert_name,
+ag.neutralization_tier,
+ag.clade,
+ag.antigen_description,
+ag.antigen_control,
+ag.virus_full_name,
+ag.virus_name_other,
+ag.virus_species,
+ag.virus_host_cell,
+ag.virus_backbone,
+ag.cds_virus_id,
+GROUP_CONCAT(DISTINCT ap.panel_name, ';') AS panel_names
```

Comment:
can panel_name itself contain ';'? if so, the ui can't split it correctly.

Reply:
Hmm good point. I am using least likely one "|" instead - what do you think?
From handful of sample data client had mentioned, it looked like a pretty straight forward text.